dbms/src/Interpreters/JoinUtils.cpp

```diff
@@ -51,6 +51,24 @@ void recordFilteredRows(
     auto column = block.getByName(filter_column).column;
     if (column->isColumnConst())
         column = column->convertToFullColumnIfConst();
+    if unlikely (column->onlyNull())
```

Comment:
Move this check before `if (column->isColumnConst())`?

Reply:
ok, updated it